staking-cli/src/bin/staking-cli.rs

```diff
@@ -157,6 +166,18 @@ pub async fn main() -> Result<()> {
         _ => {}, // Other commands handled after shared setup.
     }
 
+    // When the staking CLI is used for our testnet, the env var names are different.
+    let config = config.apply_env_var_overrides()?;
```

Comment:
why are the env var different?

Reply:
Because they are really long for the sequencer, so I thought it's weird to use those for the staking-cli.

Reply:
can we not use different long name but same env?

Reply:
I mean the env vars are long and IMO don't really make sense for the stake table CLI. But yeah we could use the same. I will do a follow up PR to address your suggestions.